src/librustc/ty/maps/on_disk_cache.rs

```diff
@@ -277,6 +280,31 @@ impl<'sess> OnDiskCache<'sess> {
                 diagnostics_index
             };
 
+            let interpret_alloc_index = {
```

Comment:
This exact code exists twice, once for `on_disk_cache` and once for `rustc_metadata::encoder`. Not sure how to dedup.